src/pages/Story.js

```diff
@@ -184,12 +215,22 @@ const Story = (props) => {
               )}
               <div className='story-buttons-container'>
                 {editMode && !editor ? (
-                  <Button
-                    className='btn btn-default'
-                    onClick={() => setEditor(true)}
-                  >
-                    Edit
-                  </Button>
+                  <>
+                    <Button
+                      className='btn btn-default'
+                      onClick={() => setEditor(true)}
+                    >
+                      Edit
+                    </Button>{' '}
+                    &nbsp; &nbsp;
+                    <object className='share-story' onClick={togglePopup}>
+                      <i className='eos-icons share-story'> share </i>
+                    </object>{' '}
+                    &nbsp; &nbsp;
+                    <object className='share-story' onClick={copy}>
```

Comment:
what's this object btw? I have never used it 👀

Reply:
Object tag is actually used for external resources. Actually I wanted to make these two icons clickable and at the same time wanted them to be in the same div as that of button that's why I used this. There must be a better way to do this.

Reply:
I have changed the object to transparent button :)